src/orion/core/worker/experiment.py

```diff
@@ -297,6 +297,18 @@ def fetch_completed_trials(self):
 
         return completed_trials
 
+    def fetch_active_trials(self):
+        """Fetch currently incomplete trials for this `Experiment`
+
+        :return: list of incomplete `Trial` objects
+        """
+        # TODO(mnoukhov): add reserved to query?
```

Comment:
Yes indeed!

Reply:
It should actually be incompleted trials in general. That means any status that is not `completed`. We want to lie for new trials and broken trials as well.